covfee/server/orm/journey.py

```diff
@@ -271,4 +271,4 @@ def to_dict(self, with_nodes=False, with_response_info=False):
         return instance_dict
 
     def make_results_dict(self):
-        return {"nodes": [node.id for node in self.nodes]}
+        return {"nodes": [node.id for node in self.nodes], "global_unique_id": self.spec.global_unique_id}
```

Comment:
are None parse well back and forth into json ?

Reply:
Surprising yes, they get converted to text `null` and then get read without issues on the conflab side.